SQL Files

jp-webdev opened the pull request:
DB creation script tested and working, and the DB drop script tested and working. DB drop script can be ran across all team machines to delete the DB, and new DB creation scripts can be easily cooked up in case we need to adjust our tables for future features.

jp-webdev committed:
Comments, and SQL Drop Script

Added comments to the SQL_Initialize.php file. Created a SQL_Drop file,
which can be used to drop the entire database, should the team need to
rebuild the database for future functionality additions.

jp-webdev committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into SQL-Files

JGefroh commented on backend/db-init/SQL_Drop.php:
```diff
@@ -0,0 +1,28 @@
+<?php
+
+// Initializing DB login variables -- compliant with the login information decided on by the team
+$database_host = 'localhost';
```
`localhost` tried to connect to a non-existent socket on my Mac. Changing this to `127.0.0.1` worked.

JGefroh commented:
Looks good except for that one comment. I'm OK with merging this when you're ready.

jp-webdev committed:
Assigning IP to DB host instead of localhost

Fixing an issue on Mac where the DB initialization script connects to a
non-existent socket. Using 127.0.0.1 instead.

jp-webdev commented:
Made necessary changes -- merging into dev now
